### PR TITLE
Convert attribute values to strings

### PR DESCRIPTION
**Pull request by choumx**
Partial for #317. Fixes 6 test failures in #319.

### CODE REVIEW

File: src/worker-thread/dom/Element.ts [hunk 1]
```diff
@@ -202,7 +202,7 @@ export class Element extends ParentNode {
    * @param name attribute name
    * @param value attribute value
    */
-  public setAttribute(name: string, value: string): void {
+  public setAttribute(name: string, value: Object | null | undefined): void {
```
Comment (jridgewell) [inline src/worker-thread/dom/Element.ts]:
```suggestion
  public setAttribute(name: string, value: any): void {
```

File: src/worker-thread/dom/Element.ts [hunk 2]
```diff
@@ -254,26 +254,27 @@ export class Element extends ParentNode {
    * @param name attribute name
    * @param value attribute value
    */
-  public setAttributeNS(namespaceURI: NamespaceURI, name: string, value: string): void {
+  public setAttributeNS(namespaceURI: NamespaceURI, name: string, value: Object | null | undefined): void {
```
Comment (jridgewell) [inline src/worker-thread/dom/Element.ts]:
```suggestion
  public setAttributeNS(namespaceURI: NamespaceURI, name: string, value: any): void {
```

File: src/worker-thread/dom/Element.ts (hunk 1 above)
Comment (jridgewell) [reply src/worker-thread/dom/Element.ts]: Or, `unknown` if we lint against `any`.

Comment (choumx) [reply src/worker-thread/dom/Element.ts]: `any` opts out of type checking, so we want `unknown` (TIL).

**Pull request merged**